tests/framework/e2e/cluster.go

```diff
@@ -881,16 +883,13 @@ func (epc *EtcdProcessCluster) StartNewProcFromConfig(ctx context.Context, tb te
 
 // UpdateProcOptions updates the options for a specific process. If no opt is set, then the config is identical
 // to the cluster.
-func (epc *EtcdProcessCluster) UpdateProcOptions(i int, tb testing.TB, opts ...EPClusterOption) error {
+func (epc *EtcdProcessCluster) UpdateProcOptions(i int, tb testing.TB, newflags ...string) error {
 	if epc.Procs[i].IsRunning() {
 		return fmt.Errorf("process %d is still running, please close it before updating its options", i)
 	}
 	cfg := *epc.Cfg
-	for _, opt := range opts {
-		opt(&cfg)
-	}
```

Comment:
are you sure this won't cause any issue?

Reply:
Don't know, but the idea of `UpdateProcOptions` function is incorrect. We should not reload config like this.